src/Jobs/NodeSubscriptorJob.cs

```diff
@@ -0,0 +1,52 @@
+using FundsManager.Data.Repositories;
+using FundsManager.Data.Repositories.Interfaces;
+using FundsManager.Helpers;
+using Quartz;
+
+namespace FundsManager.Jobs;
+
+public class NodeSubscriptorJob : IJob
+{
+    private readonly ILogger<NodeSubscriptorJob> _logger;
+    private readonly ISchedulerFactory _schedulerFactory;
+    private readonly INodeRepository _nodeRepository;
+
+    public NodeSubscriptorJob(ILogger<NodeSubscriptorJob> logger, ISchedulerFactory schedulerFactory, INodeRepository nodeRepository)
+    {
+        _logger = logger;
+        _schedulerFactory = schedulerFactory;
+        _nodeRepository = nodeRepository;
+    }
+
+    public async Task Execute(IJobExecutionContext context)
+    {
+        _logger.LogInformation("Starting {JobName}... ", nameof(NodeSubscriptorJob));
+        try
+        {
+            var managedNodes = await _nodeRepository.GetAllManagedByFundsManager();
+
+            var scheduler = await _schedulerFactory.GetScheduler();
+            
+            foreach (var managedNode in managedNodes)
+            {
+                if (managedNode.ChannelAdminMacaroon != null)
+                {
+                    var map = new JobDataMap();
+                    map.Put("nodeId", managedNode.Id.ToString());
+                    var job = RetriableJob.Create<NodeChannelSuscribeJob>(map, managedNode.Id.ToString(), new []{1,1,1,1});
```

Comment:
should this be a retriable job? Shouldn't this run forever? I think you should just restart the job like in SweepAllNodesWalletsJob, removing the "false" flag from `JobExecutionException`

Reply:
Done